static/app/views/releases/list/index.tsx

```diff
@@ -252,6 +276,22 @@ class ReleasesList extends DeprecatedAsyncView<Props, State> {
     return this.getSelectedProject()?.hasSessions ?? null;
   }
 
+  getThresholdsForRelease(release: Release): Threshold[] {
+    if (!this.hasV2ReleaseUIEnabled) {
+      return [];
+    }
+    const {thresholds} = this.state;
+    const lastDeploy = release.lastDeploy;
+    const projectSlugs = release.projects.map(p => p.slug);
+
+    return thresholds.filter(
+      threshold =>
+        projectSlugs.includes(threshold.project.slug) &&
+        (lastDeploy && lastDeploy.environment) ===
+          (threshold.environment && threshold.environment.name)
```

Comment:
does this have to be an `===`? Is this just to make sure the object goes all the way down to include `threshold.environment.name`?

Reply:
You mean `===` as opposed to `==`?
IMO we shouldn't ever really be using a loose comparison. That just leaves things sloppy :/ 

Not quite relevant here, but what i'm doing is comparing the environment names
just realized i can clean this up better to be `lastDeploy?.environment === threshold.environment?.name`

Reply:
No, I actually meant why are we doing a truthy check against two booleans, but your second comment actually makes it more clear, in that we want to compare 2 different values. Thanks for clearing that up!